sccm/core/plan-design/hierarchy/state-messaging-system-center-configuration-manager.md

```diff
@@ -30,736 +30,736 @@ Each Configuration Manager feature that uses state messages is identified by the
 
 ## 401 STATE_TOPICTYPE_SUM_CI_COMPLIANCE
 
 |     State Message ID     |  State Message Description |
 |:-------------|:------|
-|0| Compliance state unknown|
-|	1	|Compliant		   |
-|	2	|Non-compliant		   |
-|	3	|Conflict detected	   |
-|	4	|Error			   |
-|	5	|Unknown		   |
-|	6	|Partial compliance   |
-|	7	|Compliance not configured	   |
+| 0 | Compliance state unknown|
+| 1 |Compliant     |
+| 2 |Non-compliant     |
+| 3 |Conflict detected    |
+| 4 |Error      |
+| 5 |Unknown     |
+| 6 |Partial compliance   |
+| 7 |Compliance not configured    |
 
 ## 402 STATE_TOPICTYPE_SUM_CI_ENFORCEMENT
 
 |     State Message ID     |  State Message Description |
 |:-------------|:------|
-| 0 | Enforcement state unknown|
-|	1	|Enforcement started			       |
-|	2	|Enforcement waiting for content		       |
-|	3	| Waiting for another installation to complete	       |
-|	4	|Waiting for the maintenance window before installing		       |
-|	5	|Restart required before installing		       |
-|	6	|General failure	       |
-|	7	|Pending installation		       |
-|	8	|Installing update			       |
-|	9	|Pending system restart	       |
-|	10	|Successfully installed update		       |
-|	11	|Failed to install the update	       |
-|	12	|Downloading update	       |
-|	13	| Downloaded update	       |
-|	14	|Failed to download the update	       |
+| 0  | Enforcement state unknown|
+| 1  |Enforcement started          |
+| 2  |Enforcement waiting for content         |
+| 3  | Waiting for another installation to complete        |
+| 4  |Waiting for the maintenance window before installing         |
+| 5  |Restart required before installing         |
+| 6  |General failure        |
+| 7  |Pending installation         |
+| 8  |Installing update          |
+| 9  |Pending system restart        |
+| 10 |Successfully installed update         |
+| 11 |Failed to install the update        |
+| 12 |Downloading update        |
+| 13 | Downloaded update        |
+| 14 |Failed to download the update        |
 
 ## 500 STATE_TOPTCTYPE_SUM_UPDATE_DETECTION
 
 |     State Message ID     |  State Message Description |
 |:-------------|:------|
 |0 |Detection state unknown|
-|	1	| Update is not required	 |
-|	2	| Update is required		 |
-|	3	| Update is installed	 |
+| 1 | Update is not required  |
+| 2 | Update is required   |
+| 3 | Update is installed  |
 
 ## 501 STATE_TOPICTYPE_SUM_UPDATE_SOURCE_SCAN
 
 |     State Message ID     |  State Message Description |
 |:-------------|:------|
-|0 |Scan state unknown|
-|	1	| Scan is waiting for content		 |
-|	2	| Scan is running			 |
-|	3	| Scan complete			 |
-|	4	| Scan is pending retry		 |
-|	5	| Scan failed		 |
-|	6	| Scan completed with errors |
+| 0 |Scan state unknown|
+| 1 | Scan is waiting for content   |
+| 2 | Scan is running    |
+| 3 | Scan complete    |
+| 4 | Scan is pending retry   |
+| 5 | Scan failed   |
+| 6 | Scan completed with errors |
 
 ## 700 STATE_TOPICTYPE_RESYNC_STATE_MSG
 
-	No State IDs.
+No State IDs.
```

Comment:
Couldn't tell if these were intended to be rendered as code blocks, can replace with spaces if they are supposed to be blocks